### PR TITLE
Rim31/ch14/login page frontend

### PR DESCRIPTION
**Pull request by rim31**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,31 @@
...
-    <div id="App"></div>
+    <div id="App">
+		<!-- c'est ici qu'il y aura tous le react -->
+	 </div>
     <script type="text/javascript" src='js/bundle.js'></script>
   </body>
+
+
+
+	<!--[if lt IE 9]>
```
Comment (patricioguzman42) [inline index.html]: are you sure, we need this ??
i thought we needed just to run it in firefox and chrome

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+
+export default class Header extends React.Component {
+
+  render() {
+    return (
+    <header id="myHeader">
+    <div>
+    {/*<Header. {this.props.title}*/}
```
Comment (patricioguzman42) [inline src/components/Header.js]: this, should be deleted no?

File: src/components/Header/Title.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+
+export default class Title extends React.Component {
+  render() {
+    return (
+			<span>
+      {/* | Title test*/}
```
Comment (patricioguzman42) [inline src/components/Header/Title.js]: Also this

File: src/components/Welcome.js [hunk 4]
```diff
@@ -1,10 +1,35 @@
...
+    this.state = {name: "Pedro state"}
+  }
 
   render() {
+    const title="test props dans le welcome";
     return (
-        <div id='hello'>hello world</div>
+      <div id='mybody'>
+        <Header title={title}/>
+        {/*<div>Matcha Test title Maricón {this.state.name}</div>*/}
```
Comment (patricioguzman42) [inline src/components/Welcome.js]: and also this.
we should avoid keeping comments in master

**Review by patricioguzman42 — CHANGES_REQUESTED** [review]

File: src/components/Header.js (hunk 2 above)
Comment (rim31) [reply src/components/Header.js]: yes, it is just for memory

File: index.html (hunk 1 above)
Comment (rim31) [reply index.html]: no, delete

File: src/components/Welcome.js (hunk 4 above)
Comment (rim31) [reply src/components/Welcome.js]: ok

**Review by patricioguzman42 — APPROVED** [review]

**Pull request merged**